config/deploy.rb

```diff
@@ -0,0 +1,54 @@
+# config valid only for current version of Capistrano
+lock '3.4.0'
+
+set :application, 'timeoverflow'
+set :repo_url, 'git@github.com:coopdevs/timeoverflow.git'
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+# set :deploy_to, '/var/www/my_app_name'
+
+# Default value for :scm is :git
+# set :scm, :git
+
+# Default value for :format is :pretty
+# set :format, :pretty
+
+# Default value for :log_level is :debug
+# set :log_level, :debug
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+set :linked_files, fetch(:linked_files, []).push('config/database.yml', 'config/secrets.yml')
+
+# Default value for linked_dirs is []
+set :linked_dirs, fetch(:linked_dirs, []).push('log', 'tmp/pids', 'tmp/cache', 'tmp/sockets', 'vendor/bundle', 'public/system')
```

Comment:
Line is too long. [127/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.